documentation/docs/tutorials/validate_university_degree.mdx

```diff
@@ -101,25 +102,25 @@ After this step, you will find Alice's weakhold file in `./weakhold/Alice.json`.
 
 ### 2. **Issuer**: Create a DID
 
-Once you have created the Alice's DID(Decentralized Identifier), you should do the same for the University of Oslo.
+Once you have created Alice's DID (Decentralized Identifier), you will do the same for the University of Oslo.
```

Comment:
Splitting hairs here, but I don't think `will` works the same as `should` here, the user won't necessarily do anything.

Reply:
swapped back 👍